apple/internal/apple_framework_import.bzl

```diff
@@ -281,6 +281,7 @@ def _apple_dynamic_framework_import_impl(ctx):
         kind = "dynamic",
         label = label,
         libraries = [] if ctx.attr.bundle_only else framework.binary_imports,
+        swiftinterface_imports = framework.swift_interface_imports,
```

Comment:
This wasn't needed by tests, but I'm not sure if we fully test the path that would need it.